app/apollo/resolvers/subscription.js

```diff
@@ -173,19 +175,16 @@ const subscriptionResolvers = {
 
       const user = whoIs(me);
 
-      logger.debug({req_id, user, org_id, uuid, name }, `${queryName} enter`);
+      logger.debug({req_id, user, org_id, uuid, name}, `${queryName} enter`);
 
       try {
-        // await validAuth(me, org_id, ACTIONS.READ, TYPES.SUBSCRIPTION, queryName, context); // Currently uses `filterSubscriptionsToAllowed` for auth
         let subs = await subscriptionResolvers.Query.subscriptions(parent, { orgId: org_id }, context, fullQuery);
         const matchingSubs = subs.filter( s => {
           return (s.uuid === uuid || s.name === name);
         } );
-        subs = await filterSubscriptionsToAllowed(me, org_id, ACTIONS.READ, TYPES.SUBSCRIPTION, subs, context);
 
         // If more than one matching subscription found, throw an error
         if( matchingSubs.length > 1 ) {
-          logger.info({req_id, user, org_id, uuid, name }, `${queryName} found ${matchingSubs.length} matching subscriptions` );
```

Comment:
Seems to be a valuable log point, don't remove

Reply:
fixed